api/v1alpha1/istio_types.go

```diff
@@ -44,6 +44,12 @@ type IstioSpec struct {
 	// +kubebuilder:validation:Schemaless
```

Comment:
@luksa Should we remove `+kubebuilder:pruning:PreserveUnknownFields` and `+kubebuilder:validation:Schemaless` markers for `values` field?

Reply:
What's the effect? How does the OpenAPI schema change? How do these markers affect validation?

Reply:
It looks like this [tag](https://book.kubebuilder.io/reference/markers/crd-validation) `+kubebuilder:validation:Schemaless` disables all type checking. So we may want to enable this check for the type-safe helm values (it may be done from this [PR](https://github.com/maistra/istio-operator/pull/1303))

Reply:
Yes, let's remove this.